lib/enumerize/activerecord.rb

```diff
@@ -21,8 +21,14 @@ def enumerize(name, options={})
           require 'enumerize/hooks/uniqueness'
 
           unless options[:multiple]
-            decorate_attribute_type(name, :enumerize) do |subtype|
-              Type.new(enumerized_attributes[name], subtype)
+            if ::Gem::Version.new(::ActiveRecord::VERSION::STRING) >= ::Gem::Version.new("6.1.0.alpha")
```

Comment:
as far as I know you can use `::ActiveRecord.version` to get the `Gem::Version` instance

Reply:
Thanks, I fixed!